demo/server/src/app.ts

```diff
@@ -1,33 +1,112 @@
 'use strict';
 
 import * as express from 'express';
+import * as session from 'express-session';
 import * as path from 'path';
 import * as logger from 'morgan';
 import * as cookieParser from 'cookie-parser';
 import * as bodyParser from 'body-parser';
 import index from './routes/index';
 import * as favicon from 'serve-favicon';
 import EnvironmentConfig, { Config, CloudAppConfig } from './util/config';
+// import * as passport from 'passport';
+// import {Strategy} from 'passport-local';
+import users, {User} from './userSeed';
```

Comment:
Get that wrapped into separate interface which can be implemented by users/use our store or in this particular case use list of seeded users.